.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
+  printWidth: 100,
```

Comment:
Can we remove `printWidth` option here? (and leave the default value)

Reply:
The rule being followed here is https://airbnb.io/javascript/#whitespace--max-len.

Reply:
Yeah, but at the same time, 120 is the default option proposed by prettier. So far, we've used prettier's one so I think that we shouldn't change this.